README.md

```diff
@@ -68,3 +68,48 @@ Yes! Guest authors can be disabled entirely through a filter. Having the followi
 ## Change Log
 
 [View the change log](https://github.com/Automattic/Co-Authors-Plus/blob/master/CHANGELOG.md).
+
```

Comment:
Thanks for bringing this up. I also assumed it should work this way, but no other taxonomies do that. If you list the categories or tags within a post and change them, they don't change in the content until you save the post and refresh.

I believe that it works that way because the taxonomies are not saved until you save the post, and if they are not saved, they cannot be fetched using the REST API. The same is true for us using the `author` taxonomy here.

I mentioned this here during early work on the blocks:
https://github.com/Automattic/Co-Authors-Plus/issues/940#issuecomment-1652052154